developing-packages.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: page
+title: Developing Packages & Plugins
+permalink: /developing-packages/
+---
+
+* TOC
+{:toc}
+
+## Package introduction
+
+Dart packages enable the creation of modular code that can easily be shared. A
+minimal package with a single library consists of:
+
+* A `pubspec.yaml` file: A metadata file that declared the package name,
+  version, author, etc.
+
+* A `lib` directory containing the public code in the package, minimally a
+  single `<package-name>.dart` file.
+
+For additional details, see the [Dart library package](https://www.dartlang.org/guides/libraries/create-library-packages) documentation.
+
+### Package types {#types}
+
+Flutter supports several kinds of packages:
```

Comment:
as a package developer, it's still one kind of package. It can contain a few different things: dart library, native code, examples, LICENSE, etc. But it's one package format.

The details you talk about are important, I'm commenting on the phrasing. s/types of packages/kinds of things inside a package